src/test/java/bio/terra/service/profile/ProfileServiceUnitTest.java

```diff
@@ -109,37 +109,18 @@ public void testUpdateProfile() {
         .thenReturn(jobBuilder);
 
     String result = profileService.updateProfile(billingProfileUpdateModel, user);
-    verify(iamService, times(1))
-        .verifyAuthorization(
-            eq(user),
-            eq(IamResourceType.SPEND_PROFILE),
-            eq(updateId.toString()),
-            eq(IamAction.UPDATE_BILLING_ACCOUNT));
+
     verify(jobBuilder, times(1)).submit();
     assertEquals(result, jobId);
   }
 
-  @Test(expected = IamForbiddenException.class)
-  public void testUpdateProfileNoAccess() {
-    doThrow(IamForbiddenException.class)
```

Comment:
This `verifyAuthorization` check used to be inside of `profileService.updateProfile` but now it is in the profile api controller so this is now covered by `ProfileApiControllerTest.testUpdateProfileForbidden` (same with deleting a profile)